### PR TITLE
[ADD] MRP: Three-step manufacturing

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR adds a new doc detailing how to manufacture products using three steps.

NOTE: There is a ref tag in this doc that does not currently work because the doc it is referencing is not yet live. The doc being referenced should be live by the time this doc is published, at which point, the ref tag will work.

### CODE REVIEW

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+After completing the manufacturing order, return to the order's :guilabel:`Transfers` page by
+clicking the :guilabel:`Transfers` button at the top of the order. This time, select
+:guilabel:`WH/SFP/XXXXX` to open the store finished products transfer. This transfer is used to
+track the movement of finished products from the location where they were manufactured to the
+location where they are stored.
+
+After transferring the finished products to their storage location, click :guilabel:`Validate` at
+the top of the transfer, followed by :guilabel:`Apply` on the :guilabel:`Immediate Transfer?` pop-up
+window that appears. Doing so moves the transfer to the :guilabel:`Done` stage and updates inventory
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:168]:
```suggestion
the top of the transfer. Then, on the :guilabel:`Immediate Transfer?` pop-up window, click
:guilabel:`Apply`. Doing so moves the transfer to the :guilabel:`Done` stage and updates inventory
```

Optional suggestion to make the pop-up distinct from the UI before

**Review by Felicious — APPROVED** [review]
Great job, @hojo-odoo! Just finished going over the last section of the 3-step manufacturing flow. This makes a great addition to the manufacturing docs! 

Also I apologize for the two commits. I saw a merge conflict and it took two tries to resolve it :sweat_smile:

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+========================
+
+.. _manufacturing/management/two_step_manufacturing:
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
+three-step manufacturing, Odoo creates a pick components transfer, a manufacturing order (MO), and a
+store finished products transfer, and updates inventory counts based on the number of components
+removed, and finished products created.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:12]:
```suggestion
Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
three-step manufacturing, Odoo creates a pick components transfer, a manufacturing order (MO), and a
store finished products transfer.

Additionally, Odoo updates inventory counts based on the number of components removed and finished
products created.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+   components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
+   store products (3 steps)`.
+
+   .. image:: three_step_manufacturing/manufacturing-type.png
+      :align: center
+      :alt: The Manufacture radio input field on a warehouse configuration page.
+
+.. important::
+   Products must be properly configured before they can be manufactured in Odoo. For details on how
+   to do so, see the documentation on how to :ref:`configure a product for manufacturing
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:31]:
```suggestion
   to accomplish this, see the documentation on how to :ref:`configure a product for manufacturing
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+================================
+
+After confirming a three-step |MO|, a :guilabel:`Transfers` smart button appears at the top of the
+page. Click it to be taken to the :guilabel:`Transfers` page for the |MO|. The page lists two
+transfers: *WH/PC/XXXXX* (the pick components transfer), and *WH/SFP/XXXXX* (the store finished
+products transfer).
+
+Select :guilabel:`WH/PC/XXXXX` to open the pick components transfer for the |MO|. This transfer is
+used to track the movement of components from the locations where they are stored to the location
+where they are used to manufacture the product.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:65]:
```suggestion
used to track the movement of components, from the locations where they are stored to the location
where they are used to manufacture the product.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+Basic workflow
+--------------
+
+To complete work orders from the |MO| itself, begin by navigating to :menuselection:`Manufacturing
+--> Operations --> Manufacturing Orders`, then select a manufacturing order.
+
+On the |MO| page, select the :guilabel:`Work Orders` tab. Once work begins on the first work order
+that needs to be completed, click the :guilabel:`Start` button for that work order. Odoo
+*Manufacturing* then starts a timer that keeps track of how long the work order takes to complete.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:93]:
```suggestion
*Manufacturing* then starts a timer to keep track of how long the work order takes to complete.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+.. image:: three_step_manufacturing/tablet-view-button.png
+   :align: center
+   :alt: The tablet view button for a work order on a manufacturing order.
+
+After opening the tablet view, Odoo *Manufacturing* automatically starts a timer that keeps track of
+how long the work order takes to complete. After completing the work order, click the
+:guilabel:`Mark as Done` button in the top-right corner of the tablet view.
+
+Clicking :guilabel:`Mark as Done` while there is at least one more work order left to complete opens
+a page that lists the next work order. Click on that work order to open it in the tablet view.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:128]:
```suggestion
Clicking :guilabel:`Mark as Done`, while there is *at least* one more work order left to complete,
opens a page that lists the next work order. Click on that work order to open it in the tablet view.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
+button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
+:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done` and register the
+manufactured product(s) into inventory.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:133]:
```suggestion
button appears on the tablet view, in addition to the :guilabel:`Mark as Done` button. Click
:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done`, and register the
manufactured product(s) into inventory.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+It is also possible to complete the final work order while keeping the |MO| open, by clicking
+:guilabel:`Mark as Done`. In this case, the |MO| can be closed at a later time by clicking the
+:guilabel:`Produce All` button on the |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:137]:
```suggestion
It is also possible to complete the final work order, while keeping the |MO| open, by clicking
:guilabel:`Mark as Done`. In this case, the |MO| can be closed at a later time by clicking the
:guilabel:`Produce All` button on the |MO|.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+Process finished product transfer
+=================================
+
+After completing the |MO|, return to the order's :guilabel:`Transfers` page by clicking the
+:guilabel:`Transfers` button at the top of the order. This time, select :guilabel:`WH/SFP/XXXXX` to
+open the store finished products transfer. This transfer is used to track the movement of finished
+products from the location where they were manufactured to the location where they are stored.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:145]:
```suggestion
After completing the |MO|, return to the order's :guilabel:`Transfers` page by clicking the
:guilabel:`Transfers` button at the top of the order.

This time, select :guilabel:`WH/SFP/XXXXX` to open the store finished products transfer. This
transfer is used to track the movement of finished products, from the location where they were
manufactured to the location where they are stored.
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo -- another great doc. Just another tiny amount of minor modifications that will need your attention. But, once you make those adjustments, I think this one will be ready for the next stage. Nice job!

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+--------------
+
+To complete work orders from the |MO| itself, begin by navigating to :menuselection:`Manufacturing
+--> Operations --> Manufacturing Orders`, and then select a manufacturing order.
+
+On the |MO| page, select the :guilabel:`Work Orders` tab. Once work begins on the first work order
+that needs to be completed, click the :guilabel:`Start` button for that work order. Odoo
+*Manufacturing* then starts a timer that keeps track of how long the work order takes to complete.
+
+.. image:: three_step_manufacturing/start-button.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:95]: this image doesn't *appear* to be compressed I was able to get it down to about ~14kb

**Review by StraubCreative — APPROVED** [review]
@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**